contents/teams/growth/objectives.mdx

```diff
@@ -1,43 +1,33 @@
-## Q2 2025 Objectives
+## Q3 2025 Objectives
 
-### Josh
+### Joshua
 
-1. Installation & SDK experience: Automate the setup of 50% of PostHog installations
-    - Setup wizard:
-        - Support more integrations
-        - Uploading environment variables to common hosting providers
-        - PR workflow
-        - Github action
-        - Automatic event instrumentation
-    - Be the default analytics provider for vibe coders
-    - Be the default option in boilerplates for NextJS, React & React Native
-    - Improve the onboarding experience for self hosting users
+- PostHog applications
+    - Rollout OAuth support
+    - Applications beta - organizations can create applications to integrate with PostHog
+    - Make our APIs easier to use (e.g. generate API SDKs, improve OpenAPI spec etc.)
 
-2. Make it easier to use multiple products
-    - Project homepage: make this a good place to see an overview of all your products
-    - Gather common pitfalls from user onboarding calls and replays in each product and remove them
+- AI developer experience
+    - Make MCP server compatible with OpenAI, Anthropic
+    - Add functionality to the MCP server based on user feedback
+    - Add more products to the AI wizard integration
+    - Move MCP server to our own infra to support EU users
 
-3. Launch a creative product experiment once a month!
+- Side quests (if there is time):
+    - Reduce CI times
+    - Make internal retargeting easier e.g. triggering events for marketing / sales to subscribe to
```

Comment:
Nope - this is just "do the thing"

```suggestion
    - Set up basic internal retargeting experiment
```

Growth would own this completely, and it all happens within our tools. How do we identify that someone is ripe for a specific product, then point them in that direction? Initial thought I had (but it doesn't necessarily have to go this way) is just a small ad space somewhere, put a little ad for a product they aren't using but should be (based on the other products they're using), and see if we get any clicks on it.